### PR TITLE
update-avalanche-docs

### PR DESCRIPTION
**Pull request by aditya-manit**

### CODE REVIEW

File: pages/validator/external-chains/avalanche.md [hunk 1]
```diff
@@ -1,76 +1,189 @@
...
+
+# get out of binaries directory
+cd $HOME
+
+# verify version
+avalanchego --version
 ```
 
-The script will start and prompt you for information about your server environment. Follow the required steps, enter your network information, and confirm the installation. The script will then automatically create and start the `avalanchego.service`. To check if the service is running or to follow the logs, use the following commands:
+## Set environment variables
```
Comment (talalashraf) [inline pages/validator/external-chains/avalanche.md:75]: can we add a note to update zshrc or whatever tool someone is using ?

**Review by talalashraf — CHANGES_REQUESTED** [review]

File: pages/validator/external-chains/avalanche.md [hunk 2]
```diff
@@ -1,76 +1,189 @@
...
+  "result": {
+    "isBootstrapped": false # wait for it to become true
+  },
+  "id": 1
+}
 ```
 
-Example:
-`http://192.168.192.168:9650/ext/bc/C/rpc`
+## Endpoint
```
Comment (milapsheth) [inline pages/validator/external-chains/avalanche.md:185]: can you also add the vald config for it, similar to what we do for Ethereum?

Comment (aditya-manit) [reply pages/validator/external-chains/avalanche.md:185]: Sure

File: pages/validator/external-chains/avalanche.md (hunk 1 above)
Comment (aditya-manit) [reply pages/validator/external-chains/avalanche.md:76]: Added

**Comment by aditya-manit** [comment]
Apologies for late update on this PR, somehow the notification emails from this PR went to my promotion tab, I was under the impression it was still to be reviewed.

File: pages/validator/external-chains/avalanche.md [hunk 3]
```diff
@@ -1,76 +1,218 @@
...
-sudo systemctl stop avalanchego
-nano  /home/avax/.avalanchego/configs/node.json
+sudo tee <<EOF >/dev/null /etc/systemd/system/avalanchego.service
+[Unit]
+Description=Avalanche daemon
+After=network-online.target
+
+[Service]
+User=root
+ExecStart=/usr/bin/avalanchego --http-host= --network-id=$NETWORK_ID
```
Comment (milapsheth) [inline pages/validator/external-chains/avalanche.md:115]: http-host isn't needed?

**Review by milapsheth — APPROVED** [review]

**Pull request merged**